### ISSUE TITLE
Blocking my IP (cloudflare) from dns

### ISSUE THREAD

**Issue by fornacis-202** [keivanipchihagh/x-ui#11]
Hi. Recently ns lookup of my domain in Iran network results in 10.10.34.34. As I understood, this is a way of blocking my domain (by returning a false IP). Besides, changing the DNS server does not work; apparently they are blocking any A request and respond it themselves (with an unreachable IP for those they want to block). Do you have any suggestion on what I should do? I have tried creating a new domain but it took only a day to block the new one.

**Comment by keivanipchihagh** [comment]
Are you turning on the _"proxied"_ checkbox on your DNS records? If so, doing this will move your server behind Cloudflare CDNs, and the IP address you see when pinging is not your server anymore, but the Cloudflare's.

**Comment by fornacis-202** [comment]
Yes, I did that. I am  expecting Cloudflare's IP when pinging my domain, and in fact I see that correct IP when looking through Valid DNS (such as google or cloudflare). But when pinging in Iran's network, 10.10.34.34 is returned instead of Cloudflare's IP.

**Comment by keivanipchihagh** [comment]
This is strange. Let's do a little scenario. Try making two subdomains, one proxied and the other one not. Wait a little for the configurations to settle in and try pinging them from both Iran and using online tools. What results do you get?

**Comment by Nmaximillian** [comment]
@keivanipchihagh : could you tell me how you enabled the proxy in cloudflare? Cause afaik, the proxy doesn't support port 54321
So once I enabled that, I cannot access the dashboard anymore.
